src/view.rs

```diff
@@ -859,3 +859,23 @@ mod view_response_builder_tests {
         assert_eq!(expected, got);
     }
 }
+
+pub struct AllDocumentsViewValue {
```

Comment:
The json key is called `value` and `ViewResponse` has a function that is literally called `value` that returns it, so I thought it would make the most sense to use value. But I don't mind calling it response if you'd prefer that.